### PR TITLE
Add FuncRef.call_funcv

### LINKED ISSUE #31143: Add FuncRef.callv( Array arg_array ) method
**Godot version:**
3.1.1.stable.official

**Issue description:**
To call a function using `FuncRef.call_func( ... )`, you need to know in advance how many arguments it takes.

Although the `Object` class has two separate methods (`Object.call` and `Object.callv`).

`Object.call( String method, ... )`
`Object.callv( String method, Array arg_array )`

### PR DESCRIPTION
**Pull request by bojidar-bg**
Closes #31143

### CODE REVIEW

File: core/func_ref.cpp [hunk 1]
```diff
@@ -77,6 +88,8 @@ void FuncRef::_bind_methods() {
 		ClassDB::bind_vararg_method(METHOD_FLAGS_DEFAULT, "call_func", &FuncRef::call_func, mi, defargs);
 	}
 
+	ClassDB::bind_method(D_METHOD("call_funcv", "arguments"), &FuncRef::call_funcv);
```
Comment (akien-mga) [inline core/func_ref.cpp]: In Object the argument is called `arg_array`, it would be good to be consistent.

**Comment by akien-mga** [comment]
Please also add the relevant docs :)

**Comment by bojidar-bg** [comment]
Done!

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**